app/controllers/claims_controller.rb

```diff
@@ -3,8 +3,31 @@ def new
   end
 
   def create
-    TslrClaim.create!
+    claim = TslrClaim.create!
+    session[:tslr_claim_id] = claim.to_param
```

Comment:
Is there anything we should consider setting about our session cookies for the app? Is there a defualt expiry time that may not meet our needs? 

As I understand it, Rails stores the cookies securely so that is not an issue?

Reply:
Yep, so rather than implement this as part of this story I've added a post-it to the user story map specifically for this:

![IMG_3423](https://user-images.githubusercontent.com/3687/56967971-2d22e100-6b5a-11e9-902e-a988a4c2c537.jpg)

It will need planning/prioritising and probably some service design work to define the exact flow (e.g. what content do we show the user that has their session expire on them? Do we pre-warn the user about the expiring session?)